src/test/java/com/helixteamhub/plugins/jenkins/HelixTeamHubEventTest.java

```diff
@@ -19,12 +18,11 @@ public void setup() {
         String name = "test";
         String projectId = "platform";
         String repositoryId = "backend";
-        HelixTeamHubRepository repository = new HelixTeamHubRepository(projectId, repositoryId);
         String ref = "master";
         String revisionId = "d27327b131054d4a9922546aa5ecb14ba22f45ad";
         String buildUrl = "https://example.com/jenkins/job/test/42";
 
-        event = new HelixTeamHubEvent(operation, name, repository, ref, revisionId, buildUrl);
+        event = new HelixTeamHubEvent(operation, name, projectId, repositoryId, ref, revisionId, buildUrl);
```

Comment:
What if projectId and repositoryId are null when passed into HelixTeamHubEvent? Perhaps a simple test for this error condition as it would simulate a user with old data.